src/plays/photoshopApp/styles.css

```diff
@@ -0,0 +1,82 @@
+/* enter stlyes here */
+*, *::before, *::after {
+    box-sizing: border-box;
+  }
+  
+  body {
+    margin: 0;
+  }
+  
+  .container {
```

Comment:
Recommended className format is `playname-classname`

Reply:
Sure Sir..Will Update it

Reply:
Done and Committed Sir